Cargo.toml

```diff
@@ -25,7 +25,7 @@ cfg-if = "0.1"
 compiler_builtins = { version = "0.1", optional = true }
 core = { version = "1.0", optional = true, package = "rustc-std-workspace-core" }
 
-[target.'cfg(any(unix, target_os = "redox"))'.dependencies]
+[target.'cfg(unix)'.dependencies]
```

Comment:
Newer `libc` versions assume that Redox is `unix`, and since we use exactly those versions Redox support on older Rust versions will not work either way.

Reply:
seems reasonable to me